### PR TITLE
remove migrations

### LINKED ISSUE #454: Remove the custom migration logic from all runtimes
As part of our efforts to upgrade the dependencies to v0.9.42, we added custom migration logic to all runtimes. Now that these changes are rolled out and all the migrations applied, we can remove that extra logic again.

## TODO
- Remove the `CustomOnRumtimeUpgrade` implementations again
- Remove the customized dependencies. We had to create forks of some projects in order to be able to modify their migration logic. We can delete those forks again and switch back to the 'official'/upstream dependency versions.

### PR DESCRIPTION
**Pull request by b-yap**
Closes #454.

### CODE REVIEW

**Comment by b-yap** [comment]
runtime upgrade tested Amplitude chopsticks

Screenshot 2024-10-10 at 10 54 44 PM

Screenshot 2024-10-10 at 10 54 38 PM

File: runtime/amplitude/src/lib.rs [hunk 1]
```diff
@@ -158,46 +158,19 @@ parameter_types! {
...
 }
 
-// TODO remove contract migrations after update
-use pallet_contracts::migration::{v11, v12, v13, v14, v15};
-
 // Custom storage version bump
 use frame_support::{
 	pallet_prelude::StorageVersion,
 	traits::{GetStorageVersion, OnRuntimeUpgrade},
 };
```
Comment (ebma) [inline runtime/amplitude/src/lib.rs:165]: I think these imports can be removed in all runtimes.

**Review by ebma — APPROVED** [review]
Looks good to me 👍

**Review by TorstenStueber — APPROVED** [review]
👍

**Review by gianfra-t — APPROVED** [review]
Super. But let's not merge this before creating the Pendulum release for the upgrade! I think it's not yet there.

**Comment by prayagd** [comment]
@b-yap is there something pending here?

**Review by ebma — APPROVED** [review]
I think we can merge this now.

**Pull request merged**